<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>timeSettings</class>
 <widget class="QDialog" name="timeSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>211</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Study details</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Time protocol</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QRadioButton" name="rb_userDefined">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select to set the &amp;quot;Experiment duration&amp;quot; and the &amp;quot;Save frame every&amp;quot; entries manually.  &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>User defined</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="rb_studyProt">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select to use a predifined experimental protocol. Two choises are available: &lt;/p&gt;&lt;p&gt;i) Dynamic: The &amp;quot;Experiment duartion&amp;quot; is 60 min and the frames are saved every 120 sec.&lt;/p&gt;&lt;p&gt;ii) Fast dynamic:   The &amp;quot;Experiment duartion&amp;quot; is 60 min and the frames are saved from 0 to 1 min every  10 sec,  0 to 5 min every 30 sec, 5 to 10 min every 60 sec and 10 to 60 min every 120 sec&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Study protocol:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cB_expTime_2"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Experiment duration:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="cb_frameUnit">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select between units. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtCurrent</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>Save frame every:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="sB_duration">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The exprimental time. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="sB_frame">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The time step that a frame wiil be saved. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="cb_expUnit">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select between units. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="insertPolicy">
           <enum>QComboBox::InsertAtCurrent</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QPushButton" name="PB_prev">
       <property name="text">
        <string> Back</string>
       </property>
       <property name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/if_Back_32435.png</normaloff>:/icons/if_Back_32435.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="PB_ok">
       <property name="text">
        <string>Finish</string>
       </property>
       <property name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/check.png</normaloff>:/icons/check.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rb_userDefined</tabstop>
  <tabstop>rb_studyProt</tabstop>
  <tabstop>cB_expTime_2</tabstop>
  <tabstop>sB_duration</tabstop>
  <tabstop>cb_expUnit</tabstop>
  <tabstop>sB_frame</tabstop>
  <tabstop>cb_frameUnit</tabstop>
  <tabstop>PB_prev</tabstop>
  <tabstop>PB_ok</tabstop>
 </tabstops>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections/>
</ui>
